example/pubspec.lock

```diff
@@ -56,7 +56,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "1.0.1"
+    version: "1.0.2"
```

Comment:
Minor update since the api name for builder changed? 1.1.0?

Reply:
NM they are not named, good morning!

Reply:
true!